manifests/charts/istio-control/istio-discovery/values.yaml

```diff
@@ -157,15 +157,12 @@ telemetry:
       enabled: false
       logging: false
       monitoring: false
-      topology: false
```

Comment:
Will this break user who happens to specify this value?

Reply:
Good question. I don't know. @ostromart what is the proper process for deprecation of value in manifests ?

Reply:
standard API deprecation: mark as deprecated for a release, then delete. mark it as such in both values.yaml and values_types.proto. 
this is an alpha API so you're not obligated to do this. i'd still do it though because it's been alpha so long it's really de facto beta.

Reply:
OK. I think I've got it working like that now. PTAL.